TenantFile/Api/Models/Entities/ResidenceRecord.cs

```diff
@@ -7,11 +7,11 @@ namespace TenantFile.Api.Models
 {
     public class ResidenceRecord
     {
-        [Key]
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+      
         public int Id { get; set; }
-        public DateTimeOffset MoveIn { get; set; }
-        public DateTimeOffset MoveOut { get; set; }
+        //NodaTime is the recommended way to interact with Postgres date/time types
```

Comment:
I think we'll be fine for date time concerns as long as we follow the logic on this page: https://www.npgsql.org/doc/types/datetime.html

Let's go ahead and convert to DateTime, but not worry about nodatime for now.